src/Serilog.Sinks.AzureTableStorage/LoggerConfigurationAzureTableStorageWithPropertiesExtensions.cs

```diff
@@ -54,7 +52,7 @@ public static class LoggerConfigurationAzureTableStorageWithPropertiesExtensions
         /// <param name="period">The time to wait between checking for event batches.</param>
         /// <param name="additionalRowKeyPostfix">Additional postfix string that will be appended to row keys</param>
         /// <param name="keyGenerator">Generates the PartitionKey and the RowKey</param>
-        /// <param name="propagateExceptions">Whether connection issues should throw an exception; disabled by default.</param>
+        /// <param name="specificProperties">Specific properties to be added to the Table Storage</param>
```

Comment:
How about:

```xml
<param name="propertyColumns">Specific properties to be written to columns. By default, all properties will be written to columns.</param>
```

(The current name is also fine, it just seems that there's a missing connection between the properties and the columns they'll end up in.)